components/dashboard/src/Login.tsx

```diff
@@ -136,21 +153,36 @@ export function Login() {
                 <div className="flex-grow h-100 flex flex-row items-center justify-center" >
                     <div className="rounded-xl px-10 py-10 mx-auto">
                         <div className="mx-auto pb-8">
-                            <img src={gitpodIcon} className="h-16 mx-auto" alt="Gitpod's logo" />
+                            <img src={providerFromContext ? gitpod : gitpodIcon} className="h-14 mx-auto" alt="Gitpod's logo" />
                         </div>
+
                         <div className="mx-auto text-center pb-8 space-y-2">
-                            <h1 className="text-3xl">Log in{showWelcome ? '' : ' to Gitpod'}</h1>
-                            <h2 className="uppercase text-sm text-gray-400">ALWAYS READY-TO-CODE</h2>
+                            {providerFromContext
+                                ? <>
+                                    <h2 className="text-xl text-black font-semibold">Open a cloud-based development environment</h2>
+                                    <h2 className="text-xl">for the repository {repoPathname}</h2>
```

Comment:
**Optional**: I think it's more common to mention repositories as `gitpod-io/gitpod` rather than `/gitpod-io/gitpod`, so I'd slightly deviate from the specification here and remove the first slash:

```suggestion
                                    <h2 className="text-xl">for the repository {repoPathname.slice(1)}</h2>
```

Reply:
cc @jldec?

Reply:
I like having the slash, but I don't know why and can't explain it 🤣 😅 So I'll go with what @jldec approves of.

Reply:
Let's drop the leading /